libraries/compute-utils/function-views/src/rich-function-view.ts

```diff
@@ -988,7 +993,11 @@ export class RichFunctionView extends FunctionView {
       startWith(true),
       switchMap(() => {
         const newParam = this.funcCall[syncParams[field]][name];
-        return newParam.onChanged.pipe(mapTo(newParam));
+        return merge(
+          newParam.onChanged.pipe(mapTo(newParam)),
+          ...newParam.property.propertyType === DG.TYPE.DATA_FRAME && newParam.value ?
```

Comment:
This is used to trigger function run when input grid is mutated.
It could be both grid or OutliersViewer, for example.

`sub3` is not applicable, it is not called if historical run is loaded

Reply:
More accurately: sub3 is not triggered on DFs from historical runs